src/plugins/intel_gpu/CMakeLists.txt

```diff
@@ -47,10 +47,6 @@ add_subdirectory(src/graph)
 
 file(GLOB_RECURSE PLUGIN_SOURCES ${CMAKE_CURRENT_SOURCE_DIR}/src/plugin/*.cpp ${CMAKE_CURRENT_SOURCE_DIR}/include/intel_gpu/plugin/*.hpp)
 
-if(ENABLE_PROXY)
```

Comment:
Why you remove that instead of setting ENABLE_PROXY to OFF here https://github.com/openvinotoolkit/openvino/blob/master/cmake/features.cmake#L9 ?

Reply:
because in this case whole story will be disabled and PROXY plugin code can easily obsolete
with removal of GPU / NVIDIA configurations, we keep proxy compilation and tests running